.github/workflows/run_tartan.yml

```diff
@@ -0,0 +1,98 @@
+# Build, Run and Test Tartan EchoSystem
+name: Build and Run Tartan
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/run_tartan.yml'
```

Comment:
Given how long it takes, i think you should limit to workflow_dispatch

Reply:
Yes, once I am done testing, it'll only run in the workflow_dispatch